Make OnTime use the correct name of the installed newbin
--- a/ontime-electron.desktop
+++ b/ontime-electron.desktop
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Name=ontime
-Exec=AppRun --no-sandbox %U
+Exec=ontime-bin
 Terminal=false
 Type=Application
 Icon=ontime-electron
